api-reference/10 UI Widgets/dxPopover/1 Configuration/shading.md

```diff
@@ -3,3 +3,4 @@ id: dxPopover.Options.shading
 type: Boolean
 default: false
 ---
+Shading applies to the element specified by the [position](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position).[of](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) option. If this element is not specified, shading applies to the **window**.
```

Comment:
In the [position](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position
) topic we write that the position.of is connected with the target option. The latter has higher priority
@ksercs  should we mention both options here?

Reply:
Maybe, it's good idea to write that we are talking about `position.of` of popover root element (not target)

Reply:
Popover shading is not connected with `target` option/element despite what is written in the documentation